rlay-client-lib/README.md

```diff
@@ -0,0 +1,99 @@
+# Rlay Client Library
+
+Provides a high-level interface (ORM) for your application, based on your schema.
+
+This library exposes `Rlay` native `Entities` such as `Rlay_Class`, `Rlay_ClassAssertion`, `Rlay_Individual`, etc. through the client. When the client is generated and constructed with `rlay-generated`, the client will also expose the application specific `Entities` e.g. `HTTPRequestClass`, etc. These application specific `Entities` can
+
+## Usage
+
+### `rlay-generate`
+
+The default behavior for using the Rlay Client Library is, by using `rlay-generate`, which
+creates `generated/rlay-client/index.js` in the root of the project. `generated/rlay-client/index.js` exposes the initialized `Client` instance, ready for usage.
+
+```javascript
+const client = require('/generated/rlay-client');
```

Comment:
`'/generated/rlay-client'` -> `'./generated/rlay-client'`

Same with the other ones